upgrade/upgrade-scripts/execute

```diff
@@ -453,6 +468,14 @@ dpkg-query -Wf '${Conffiles}\n' | awk '$3 == "obsolete" {print $1}' |
 			die "failed to reinstall package '$package'"
 	done || die "failed to remove obsolete package configuration files"
 
+#
+# Now that all the packages have been installed, delete the newly created
+# central /etc/apt/sources.list.
+#
+if [[ -f /etc/apt/sources.list ]]; then
+	rm /etc/apt/sources.list || die "failed to delete /etc/apt/sources.list"
+fi
+
```

Comment:
Don't you need to repeat exactly what you did before in deleting both the sources.list and the `.d` directory (e.g. in case a package inadvertently installed a package source).

Reply:
I figured the next upgrade would take care of it, I've added it now.